src/components/TopBar/AboutDialog.js

```diff
@@ -0,0 +1,356 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v.2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import {
+    Alert,
+    AlertTitle,
+    Avatar,
+    Box,
+    Button,
+    Card,
+    CardContent,
+    CardHeader,
+    Chip,
+    CircularProgress,
+    Collapse,
+    Dialog,
+    DialogActions,
+    DialogContent,
+    DialogTitle,
+    Fade,
+    Grid,
+    Skeleton,
+    Tooltip,
+    useMediaQuery,
+    useTheme,
+} from '@mui/material';
+import {
+    Bookmark,
+    BrokenImage,
+    DnsOutlined,
+    Gavel,
+    QuestionMark,
+    WidgetsOutlined,
+} from '@mui/icons-material';
+import { FormattedMessage, useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+
+const ComponentsTypeIcons = {
+    app: <WidgetsOutlined />,
+    server: <DnsOutlined />,
+    other: <QuestionMark />,
+};
+
+const AboutDialog = ({
+    open,
+    onClose,
+    getGlobalVersion,
+    getLogoThemed,
+    appVersion,
+    appLicense,
+    getAdditionalComponents,
+}) => {
+    const theme = useTheme();
+    const intl = useIntl();
```

Comment:
I thnik we could use de FormattedMessage component here instead of this hook which is used for more specific internationalization management.